src/Swarm/Game/Values.hs

```diff
@@ -0,0 +1,35 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
+-- |
+-- SPDX-License-Identifier: BSD-3-Clause
+module Swarm.Game.Values where
+
+import Control.Lens (view)
+import Data.Int (Int32)
+import Linear (V2 (..))
+import Swarm.Game.Entity
+import Swarm.Game.Robot
+import Swarm.Language.Value
+
+class Valuable a where
```

Comment:
Can we add a Haddock comment to the `Value` class explaining its purpose?  I can see what it's for but I'm thinking from the perspective of potential contributors who are trying to understand the codebase.

Reply:
done